app/templates/components/doi-subjects.hbs

```diff
@@ -0,0 +1,16 @@
+<div class="form-group {{validationClass}}">
+  <label class="control-label col-md-3">Subject(s)</label>
+  <div class="col-md-9">
+    {{#each model.subjects as |subject index|}}
+      {{#if (gt index 0)}}
```

Comment:
Unless I'm missing something here, is this code actually doing anything? :)
The render looks the same.